http/http.go

```diff
@@ -33,6 +34,14 @@ var (
 	Guard RequestGuard
 )
 
+// Encodings for form data.
+//
+// See: https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/POST
+const (
+	FormEncodingMultipart = "multipart/form-data"
+	FormEncodingURL       = "application/x-www-form-urlencoded"
```

Comment:
Any reason these need to be exported? I'd prefer to keep them unexported unless we have a clear need. If there _is_ a clear need, we'd need comments on each constant to pass `golint`.

Reply:
Nope, no reason. Will fix.